harness/determined/pytorch/_load.py

```diff
@@ -112,3 +112,39 @@ def load_trial_from_checkpoint_path(path: str, **kwargs: Any) -> pytorch.PyTorch
     for idx, model in enumerate(trial_context.models):
         model.load_state_dict(checkpoint["models_state_dict"][idx])
     return trial
+
+
+def _load_pytorch_trial_for_checkpoint_export(
+    context_dir: pathlib.Path,
+    managed_training: bool,
+    trial_cls_spec: str,
+    config: Dict[str, Any],
+    hparams: Dict[str, Any],
+) -> Tuple[pytorch.PyTorchTrial, pytorch.PyTorchTrialContext]:
+    with det._local_execution_manager(context_dir):
+        trial_class = cast(pytorch.PyTorchTrial, load.trial_class_from_entrypoint(trial_cls_spec))
+        core_context, env = det._make_local_execution_env(
```

Comment:
Write this without `env`.

As-is, you're creating a big dummy object (which makes this code's behavior hard to reason about) then pulling values out of that dummy object (like env.container_gpus) which are just hardcoded values in the dummy object anyway.

Reply:
done